ir/module.go

```diff
@@ -187,6 +188,160 @@ func (m *Module) String() string {
 	return buf.String()
 }
 
+type ioWriteWrapping struct {
+	size int
+	err  error
+}
+
+func (i *ioWriteWrapping) Fprintln(w io.Writer, a ...interface{}) {
+	if i.err != nil {
+		n, err := fmt.Fprintln(w, a)
```

Comment:
Also noticed now with `go vet` that this line should include `a...`

```
./module.go:205:13: missing ... in args forwarded to print-like function
```

```diff
- 		n, err := fmt.Fprintln(w, a) 
+ 		n, err := fmt.Fprintln(w, a...) 
```

Reply:
forgot the test, I'm fixing it